SOLIDHomework.Core/Services/PaymentService.cs

```diff
@@ -0,0 +1,43 @@
+﻿using SOLIDHomework.Core.Model;
+using SOLIDHomework.Core.Payment;
+using System;
+using System.Collections.Generic;
+using System.Configuration;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SOLIDHomework.Core.Services
+{
+    public class PaymentService
+    {
+        public void Charge(PaymentDetails paymentDetails, ShoppingCart cart)
+        {
+            PaymentServiceType paymentServiceType;
+            Enum.TryParse(ConfigurationManager.AppSettings["paymentType"], out paymentServiceType);
```

Comment:
reading a configuration should be done in dependency injection container and here you will ict configuration in the constructor